wcc-contentful-app/lib/wcc/contentful/app.rb

```diff
@@ -22,8 +22,12 @@ def self.configure
       raise InitializationError, 'Cannot configure after initialization'
     end
 
-    @configuration ||= Configuration.new
-    yield(configuration)
+    WCC::Contentful.configure do |wcc_contentful_config|
+      if @configuration&.wcc_contentful_config != wcc_contentful_config
+        @configuration = Configuration.new(wcc_contentful_config)
+      end
+      yield(configuration)
```

Comment:
why would we not just pass the `WCC::Contentful` config into the passed block when configuring directly below this (middleware)?

Reply:
So that we can add attributes to WCC::Contentful::App::Configuration.  One that I'd like to add is the preview password, currently it just assumes it's in an environment variable.

Reply:
K - that makes sense if we do have something.. because there isn't anything here yet